h/sentry.py

```diff
@@ -31,6 +31,11 @@ def user_context_data(request):
         'ip_address': request.client_addr,
     }
 
+def extra_context_data(request):
```

Comment:
Not sure what this is doing here. Does it do anything or was it just for testing?